src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -428,35 +430,50 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // - Scroll the visible viewport in response to a mouse wheel event.
     // Arguments:
     // - mouseDelta: the mouse wheel delta that triggered this event.
-    // - point: the location of the mouse during this event
+    // - pixelPosition: the location of the mouse during this event

```

Comment:
```suggestion
    // - pixelPosition: the pixel location of the mouse during this event
```

Reply:
eh. i hate when the name is obvious and we are forced to document the obvious name